Src/HALALMock/Services/Time/Time.cpp

```diff
@@ -1,445 +1,168 @@
 /*
  * Time.cpp
  *
- *  Created on: 30 oct. 2022
- *      Author: Dani
+ *  Created on: 23 oct. 2024
+ *      Author: Stefan Costea
  */
 
-#include "Time/Time.hpp"
+#include "Inc/HALALMock/Services/Time/Time.hpp"
```

Comment:
This should be just
```suggestion
#include "HALALMock/Services/Time/Time.hpp"
```

Reply:
That path did not work for me, there should be a setup.env file in the workspace that sets all the necessary paths to support that and other setup actions